documentation/docs/examples/clipping.mdx

```diff
@@ -116,13 +116,15 @@ controls.addEventListener('drag', () => {
 
 // When we are hovering over a sphere we want to disable viewer controls
 controls.addEventListener('hoveron', () => { 
+  controls.activate();
   viewer.cameraControlsEnabled = false;
   draggerMaterial.color.set('gray');
   viewer.requestRedraw();
 });
 
 // And turn them back on when we stop hovering over it
 controls.addEventListener('hoveroff', () => {
+  controls.deactivate();
```

Comment:
After controls deactivation (i.e. after one drug or just 'hoveroff') you can't drug ball again and 'hoveron' event is not thrown because controls are deactivated. So we should avoid using `controls.deactivate()` I think. 
I fixed it last time when it was a different problem, not sure what broke it. Possibly the reason for controls block is inside CameraManager/ComboControls.

Reply:
Any idea how to fix the issue properly @Savokr ?

Reply:
Thinking on it now. Tried to find a commit after which it was broken